contracts/WildcardSteward_v3.sol

```diff
@@ -231,17 +231,14 @@ contract WildcardSteward_v3 is Initializable {
         uint256[] memory tokens,
         address payable[] memory _benefactors,
         uint256[] memory _patronageNumerator,
-        // uint256[] memory _tokenGenerationRate,
         address[] memory _artists,
         uint256[] memory _artistCommission,
         uint256[] memory _releaseDate
     ) public onlyAdmin {
         assert(tokens.length == _benefactors.length);
         assert(tokens.length == _patronageNumerator.length);
-        // assert(tokens.length == _tokenGenerationRate.length);
-        assert(tokens.length == _artists.length);
-        assert(tokens.length == _artistCommission.length);
         assert(tokens.length == _releaseDate.length);
+        assert(_artists.length == _artistCommission.length);
 
```

Comment:
assert(_artists.length == _artistCommission.length); 
This can be true but the lengths of these can be different to tokens.length right? We need to check for the condition too surely?